src/App.tsx

```diff
@@ -34,28 +30,25 @@ const App = () => {
 
   return (
     <div>
-      <ButtonWithInput valueFromInput={sumCountAndInputValue} />
+      <ButtonWithInput valueFromInput={valueFromInput} />
       <Count count={count} />
-      <Button text={"+1"} setFunction={plusCount} />
+      <Button text={"+1"} setFunction={valueFromInput} />
       <Button text={"-1"} setFunction={minusCount} />
```

Comment:
```suggestion
      <Button text="-1" setFunction={minusCount} />
```
Текстовые значение в пропсах необязательно оборачивать в фигурные скобки